### PR TITLE
bpo-43149: Improve error message for exception group without parentheses

### PR DESCRIPTION
**Pull request by pablogsal**
https://bugs.python.org/issue43149

### CODE REVIEW

**Review by lysnikolaou — APPROVED** [review]
LGTM!

File: Lib/test/test_syntax.py [hunk 1]
```diff
@@ -835,6 +835,39 @@
      ...
    SyntaxError: invalid syntax
 
+Check that an exception group with missing parentheses
```
Comment (graingert) [inline Lib/test/test_syntax.py:838]: This terminology conflicts with PEP 654 exception groups

cc @iritkatriel

Comment (pablogsal) [reply Lib/test/test_syntax.py:838]: Thanks for pointing that out. If exceptions group pep is merged we will surely change this sentence.

On the other hand, do you have some alternative phasing for the error you think would be clearer?

Comment (graingert) [reply Lib/test/test_syntax.py:838]: isinstance calls this thing a classinfo

Comment (graingert) [reply Lib/test/test_syntax.py:838]: https://docs.python.org/3.10/library/functions.html#isinstance

Comment (pablogsal) [reply Lib/test/test_syntax.py:838]: Unfortunately I find that even more confusing :(

Comment (iritkatriel) [reply Lib/test/test_syntax.py:838]: exception sequence?

Comment (iritkatriel) [reply Lib/test/test_syntax.py:838]: "multiple exception types"?

Comment (graingert) [reply Lib/test/test_syntax.py:838]: I started a mailing list thread in python-dev@python.org "[a name for the ExceptHandler.type when it is a literal tuple of types"](https://mail.python.org/archives/list/python-dev@python.org/message/HSN6ESRB4BD6IUIPKLMNP4TPBQPWHBFK/)

Comment (graingert) [reply Lib/test/test_syntax.py:838]: @iritkatriel  yeah I like it: `SyntaxError: multiple exception types must be parenthesized`

Comment (pablogsal) [reply Lib/test/test_syntax.py:838]: Opened https://github.com/python/cpython/pull/25996

**Pull request merged**